packages/omg-js-util/src/typedData.js

```diff
@@ -15,6 +15,7 @@
 */
 
 const NULL_ADDRESS = '0x0000000000000000000000000000000000000000'
+const OMG_ADDRESS = '0xd26114cd6ee289accf82350c8d8487fedb8a0c07'
```

Comment:
Not sure if this is the right place for this - it has nothing to do with typedData. How about moving it into transaction.js?

Also, this address is only correct on mainnet. Maybe add a comment explaining that

Reply:
yea could even seperate it out to its own currency module so we can add other specific addresses if we want

or even further:
```
export {
  NULL_ADDRESS: '0x0000000000000000000000000000000000000000',
  OMG_ADDRESS: {
    MAINNET: 0x001,
    ROPSTEN: 0x002,
    etc...
  }
}
```

Reply:
@nicholasmueller currently most tests have dependencies on transaction.ETH_CURRENCY, is it a good idea to move those over to address.ETH_ADDRESS instead? It would be a breaking change for clients relying on transaction module, but it should be a small change.

Reply:
yea i think thats fine and an easy one for clients to update when they upgrade